app/collins/models/AssetMeta.scala

```diff
@@ -206,9 +206,15 @@ object AssetMeta extends Schema with AnormAdapter[AssetMeta] with AssetMetaKeys
     val BaseProduct = findOrCreateFromName("BASE_PRODUCT")
     val BaseVendor = findOrCreateFromName("BASE_VENDOR")
     val BaseSerial = findOrCreateFromName("BASE_SERIAL")
+    val GpuCount = findOrCreateFromName("GPU_COUNT")
+    val GpuDescription = findOrCreateFromName("GPU_DESCRIPTION")
```

Comment:
I think we should have both vendor and description? I believe pci enumeration has fields for both vendor and description?